include/tbl/vsl_tags.h

```diff
@@ -107,27 +107,31 @@ SLTM(SessClose, 0, "Client connection closed",
 SLTM(BackendOpen, 0, "Backend connection opened",
 	"Logged when a new backend connection is opened.\n\n"
 	"The format is::\n\n"
-	"\t%d %s %s %s %s %s %s\n"
-	"\t|  |  |  |  |  |  |\n"
-	"\t|  |  |  |  |  |  +- \"connect\" or \"reuse\"\n"
-	"\t|  |  |  |  |  +---- Local port\n"
-	"\t|  |  |  |  +------- Local address\n"
-	"\t|  |  |  +---------- Remote port\n"
-	"\t|  |  +------------- Remote address\n"
-	"\t|  +---------------- Backend display name\n"
-	"\t+------------------- Connection file descriptor\n"
+	"\t%d %s %s %s %s %s %s %.6f %ld\n"
+	"\t|  |  |  |  |  |  |  |    |\n"
+	"\t|  |  |  |  |  |  |  |    +- Backend reuses\n"
+	"\t|  |  |  |  |  |  |  +------ Backend age in seconds\n"
+	"\t|  |  |  |  |  |  +--------- \"connect\" or \"reuse\"\n"
+	"\t|  |  |  |  |  +------------ Local port\n"
+	"\t|  |  |  |  +--------------- Local address\n"
+	"\t|  |  |  +------------------ Remote port\n"
+	"\t|  |  +--------------------- Remote address\n"
+	"\t|  +------------------------ Backend display name\n"
+	"\t+--------------------------- Connection file descriptor\n"
 	"\n"
 )
 
 SLTM(BackendClose, 0, "Backend connection closed",
 	"Logged when a backend connection is closed.\n\n"
 	"The format is::\n\n"
-	"\t%d %s %s [ %s ]\n"
-	"\t|  |  |    |\n"
-	"\t|  |  |    +- Optional reason\n"
-	"\t|  |  +------ \"close\" or \"recycle\"\n"
-	"\t|  +--------- Backend display name\n"
-	"\t+------------ Connection file descriptor\n"
+	"\t%d %s %s %.6f %ld [ %s ]\n"
+	"\t|  |  |  |    |     |\n"
+	"\t|  |  |  |    |     +- Optional reason\n"
+	"\t|  |  |  |    +------- Backend reuses\n"
+	"\t|  |  |  +------------ Backend age in seconds\n"
+	"\t|  |  +--------------- \"close\" or \"recycle\"\n"
+	"\t|  +------------------ Backend display name\n"
+	"\t+--------------------- Connection file descriptor\n"
```

Comment:
This is a breaking change, and since we are adding the same fields to two tags, maybe we should limit this to `BackendOpen`? We can deduce the values for `BackendClose` anyway based on whether we created a brand new connection or reused one from the pool.